tests/fixups/TraceFixup/CreateProcessFixup.cpp

```diff
@@ -81,13 +120,53 @@ BOOL __stdcall CreateProcessAsUserFixup(
     _In_ startupinfo_t<CharT> startupInfo,
     _Out_ LPPROCESS_INFORMATION processInformation)
 {
-    auto entry = LogFunctionEntry();
+	LARGE_INTEGER TickStart, TickEnd;
+	QueryPerformanceCounter(&TickStart);    		auto entry = LogFunctionEntry();
     auto result = CreateProcessAsUserImpl(token, applicationName, commandLine, processAttributes, threadAttributes, inheritHandles, creationFlags, environment, currentDirectory, startupInfo, processInformation);
+	QueryPerformanceCounter(&TickEnd);
     preserve_last_error preserveError;
 
     auto functionResult = from_win32_bool(result);
     if (auto lock = acquire_output_lock(function_type::process_and_thread, functionResult))
     {
+		if (output_method == trace_method::eventlog)
+		{
+			std::string inputs = "Application Name=" + InterpretStringA(applicationName) +
+				"\nCommand Line=" + InterpretStringA(commandLine);
+			std::string outputs = "Working Directory=" + InterpretStringA(currentDirectory)
+					+ "\nInheritHandles-" + bool_to_string(inheritHandles)
+					+ "\n" + InterpretProcessCreationFlags(creationFlags);
+			std::string results = "";
+			if (processAttributes)
+				outputs += "\nProcessAttributes present."; // cheap way out for now.
+			if (environment)
+			{
+				outputs += "\nEnvironment:";
+				for (auto ptr = reinterpret_cast<const CharT*>(environment); *ptr; ptr += std::char_traits<CharT>::length(ptr) + 1)
+				{
+					outputs += "\n\t" + InterpretStringA(ptr);
+				}
+			}
+			std::ostringstream sout1;
+			sout1 << "\nToken=0x" << std::uppercase << std::setfill('0') << std::setw(16) << std::hex << token;
+			outputs += sout1.str();
+
+			results = InterpretReturn(functionResult, result).c_str();
+			if (function_failed(functionResult))
+			{
+				outputs += "\n" + InterpretLastError();
+			}
+
+			std::ostringstream sout;
+			InterpretCallingModulePart1()
+				sout << InterpretCallingModulePart2()
+				InterpretCallingModulePart3()
+				std::string cm = sout.str();
+
+			Log_ETW_PostMsgOperationA("CreateProcessAsUser", inputs.c_str(), results.c_str(), outputs.c_str(),cm.c_str(), TickStart, TickEnd);
+		}
+		else
+		{
         Log("CreateProcessAsUser:\n");
         if (applicationName) LogString("Application Name", applicationName);

```

Comment:
the original path blocks should all be indented

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28